pkg/proto/datadog/remoteagent/remoteagent.proto

```diff
@@ -72,3 +72,15 @@ message GetFlareFilesResponse {
   // `.yaml` for YAML data.
   map<string, bytes> files = 1;
 }
+
+message GetTelemetryRequest {}
+
+message GetTelemetryResponse {
+  // Telemetry payload.
+  oneof payload {
+    // Prometheus "text" exposition format.
+    //
+    // https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format
+    string prom_text = 1;
```

Comment:
I'm not very familiar with the prometheus text exposition format but from my observation with this polling approach we're expecting several metrics bundled together in this response - do we have expectations/concerns around message length? I see the [this grpc client has configured it](https://github.com/DataDog/datadog-agent/pull/30543/files#diff-62826caa101a7872ffae0270f7e77f55abf48945d1ca5abde6429b94a0c183e6R358) higher than the 4MB default for flares and that is probably enough with the polling interval, it might be worth making that limit configurable as we [do the grpc server](https://github.com/DataDog/datadog-agent/blob/main/comp/api/grpcserver/impl-agent/grpc.go#L87-L88) to have a knob to turn there as we build this functionality out